src/main/java/com/bento909/model/DateMaker.java

```diff
@@ -0,0 +1,49 @@
+package com.bento909.model;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Random;
+import java.text.ParseException;
+
+public class DateMaker {
+    // Generates a date at random
+    public Date dateMaker() {
+        //ArrayList<> month31Days = new List<1,3,5,7,8,10,12>;
+        Date date;
+
+        try {
+            DateFormat df = new SimpleDateFormat("dd-MM-yyyy");
```

Comment:
This object is always the same. It would make sense to have it as a constant so it is only created once, rather than creating it every time you call the method. Doing it as I described reduces object creation which requires extra cpu, memory etc.